### DISCUSSION TITLE
Right Click context menu

### DISCUSSION THREAD

**Discussion by frenzureta** [moudey/Shell#330] [Q&A]
Hello, i recently downloaded this and edited out some context that i dont need. My problem is. There an empty space below "properties" when i right click on an item. Is there something that i removed that made an empty space on the contenxt menu?

[Click here](https://github.com/moudey/Shell/assets/101829852/c4a7e0f6-7b05-46a1-be9f-25c0ecd04b49)

**Comment by RubicBG** [comment]
https://github.com/moudey/Shell/discussions/305
